lib/wanda_web/schemas/check_execution.ex

```diff
@@ -0,0 +1,20 @@
+defmodule WandaWeb.Schemas.CheckExecution do
+  @moduledoc """
+  Check execution API spec
+  """
+
+  require OpenApiSpex
+
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Check execution",
+    description: "A single execution of checks",
+    type: :object,
+    properties: %{
+      execution_id: %Schema{type: :string, description: "Execution ID"},
+      group_id: %Schema{type: :string, description: "Group ID"},
+      payload: %Schema{type: :object, description: "Payload of the check execution"}
```

Comment:
Probably you already thought of this.

What is the plan about the shape of the payload `object`?
As an API consumer, even if at the moment is just us :smile:, I would find it useful to know that shape.

Reply:
Does it have a fixed shape? I thought it was something that can really vary

Reply:
Well, it does have a shape, and that shape is `Wanda.Execution.Result` without `:execution_id` and `:group_id`.

> Does it have a fixed shape? I thought it was something that can really vary

Until we change it :smile: 

The point to me is more 
_how stable do we think it is in relation to the needs?_
_What **significant** changes do we foresee to that representation in the upcoming iterations?_

However, I understand it might be a tedious job, moreover if things change so fast that updating the documentation becomes a burden, yet if that happens there might be another kind of issue to tackle (coupling for instance, but let's leave it out)

Bottom line: if not planned to add that missing piece in this or following PRs, then I'd find it wise to track debt.
How does it sound?

Reply:
Sounds good, I collected a followup task to better define that shape